docs/docs/guides/architecture-scenarios/b2b.mdx

```diff
@@ -0,0 +1,85 @@
+
+- **Dimitri:** a team leader who is employed by Pentagon, an Octagon department. Dimitri uses his Microsoft Account in combination with his one time password to access the portal. Pentagon therefore has set up Microsoft as Identity Provider. Pentagon also requires its users to secure their accounts with additional factor.
+- **Michael:** a trainee of Pentagon only using the portal to access his workspace apps. Michael uses his Google Account in combination with his laptops fingerprint.
+- **Bill:** is employed at Octagon as Administrator of the Portal Application. Bill too uses a Microsoft Account in combination with a Security Key to secure his account.
+
+After having determined the constellation of the organizations and its users, all the necessary data (Portal project with roles and app, users, login requirements, identity providers, branding) should be set up in [Console](https://console.zitadel.ch/org).
+A B2B sample application for NextJS can be found in our [Example Repo](https://github.com/caos/zitadel-examples).
+
+To allow an other organization to use a project, a project grant has to be created. Upon creation, roles for a grant can be limited to a subset of the total project roles.
+
+In our scenario, Octagon creates a project grant for Pentagon. Pentagon is limited to use `writer` and `reader` role. The `admin` role is reserved for the Octagon organization itself.
+
+<B2B></B2B>
+
+### Roles
+
+In this scenario, Dimitri and Michael share the same organization Pentagon, where as Bill belongs to Octagon. Octagon is owner of the Portal project with its Web App, having Bill configured as user grant with `admin` role. Dimitri owns the `writer` role, whereas Michael only is `reader`.
+
+> Note: Roles are meant for internal business logic and therefore need to be validated separately, none of the users described are allowed to create user grants, at least if they do not own a ZITADEL manager role.
+
+If you made a dashboard where some users are able to create user grants, the Management API to do such operations should be triggered with the personal access token of the users, not with a token of a machine user, to create a meaningful audit log. 
+If you had such a use case, ZITADEL manager roles must be assigned to those users.
+
+### Noteworthy
+
+Due to the fact that ZITADEL includes unlimited users, projects, and applications and comes with all security features in the FREE tier, ZITADEL can be considered a great alternative to other SaaS IAM systems such as Auth0 or Okta.
+In such a case with this high potential of scalability where user counts can grow explosively, ZITADEL does not become the bottleneck and therefore is the valid choice. You can learn more on ZITADELs benefits and the pricing [here](https://zitadel.ch/pricing).
+
+### Learn more
+
+- [Creating an organization](../basics/organizations#exercise---create-a-new-organization)
+- [Organization Branding](../customization/branding)
```

Comment:
Why do we call it branding in the docs? Everywhere else this is called privatelabeling

Reply:
Good point.. I prefer branding over privatelabeling but then it would make sense to change it in console too  🤔

Reply:
IMO we should consider switching to `branding` it sounds more natural then `privatelabelling` but yes, we need to be consistent 😁